front/pages/api/w/[wId]/labs/transcripts/index.ts

```diff
@@ -80,7 +98,14 @@ async function handler(
         });
       }
 
-      const { connectionId, provider } = bodyValidation.right;
+      const validatedBody = bodyValidation.right;
+      const connectionId =
```

Comment:
Let's have some proper typeguards here.

Reply:
Not sure what you mean there, it's already typeguarded - we need at least one of connectionId or apiKey which is enforced by if (isLeft(bodyValidation))  above